### ISSUE TITLE
SSDLite arch search cfg settings mismatch between the code and the paper

### ISSUE THREAD

**Issue by Hrayo712** [JaminFong/FNA#16]
Dear @JaminFong 

I hope you're doing well. Once again, thanks for your great work. 

I am going through the architecture search process for SSDLite. However, I noticed the following mismatches:"

- Architecture optimization in the code starts at 15 epoch, in the paper, it says epoch 12
- Lambda in the paper says 0.2, on the code is 0.15
- Batch size is reported as 64 (so 8 imgs_per_gpu), but is configured as 16 (so 128 in total)
- Total epochs are reported as 24, but configured as 30
- learning rate is mentioned to warmup from 0 to 0.03, but is configured to warmup to 0.05 instead
- Also, I see that the workflow settings for SSDLite are to alternate as: 1 iteration as arch. 5 as train. 

My question is: Should I stick to the code or what is reported on the paper ?

Thanks in advance!

**Comment by JaminFong** [comment]
Hi, it is better to refer to the settings in the code. Sorry for this inconsistency.

**Comment by Hrayo712** [comment]
Thanks for the clarification. Closing the Issue now :)
